config/locales/zh-TW.yml

```
@@ -178,121 +185,124 @@ zh-TW:
-        looking_for_maintainers:
+        looking_for_maintainers: 徵求維護者
       header:
-        dashboard: 控制台
-        settings:
-        edit_profile:
+        dashboard: 儀表板
+        settings: 設定
+        edit_profile: 編輯個人檔案
         search_gem_html: 搜尋 Gems&hellip;
         sign_in: 登入
         sign_out: 登出
         sign_up: 註冊
-      mfa_banner_html:
+      mfa_banner_html: 🎉 我們現在支援安全裝置了！<a href="/settings/edit#security-device">設定</a>新的裝置來提升您的帳號安全。[了解詳情](部落格文章連結)！
   mailer:
     confirm_your_email:
     confirmation_subject:
-    link_expiration_explanation_html:
+    link_expiration_explanation_html: 請注意，此連結將在 3 小時後過期。您可以前往<a href='https://rubygems.org/email_confirmations/new'>確認信</a>頁面要求傳送新連結。
     email_confirmation:
-      title:
-      subtitle:
-      confirmation_link:
-      welcome_message:
+      title: 電子郵件地址確認
+      subtitle: 就快完成了！
+      confirmation_link: 確認電子郵件地址
+      welcome_message: 歡迎來到 RubyGems.org！點擊下方的連結來驗證您的電子郵件地址。
     email_reset:
-      title:
-      subtitle:
+      title: 電子郵件地址重設
+      subtitle: 嗨 %{handle}！
       visit_link_instructions:
     deletion_complete:
-      title:
-      subtitle:
-      subject:
+      title: 刪除完成
+      subtitle: 掰掰！
+      subject: 您在 rubygems.org 上的帳號已被刪除
       body_html:
     deletion_failed:
-      title:
-      subtitle:
-      subject:
-      body_html:
+      title: 刪除失敗
+      subtitle: 抱歉！
+      subject: 您在 rubygems.org 上的帳號刪除請求失敗
+      body_html: 您曾在 rubygems.org 上送出帳號刪除的請求。很遺憾，我們無法處理您的請求，請稍後再試。若無法解決，請 %{contact} 我們。
     notifiers_changed:
-      subject:
-      title:
-      subtitle:
-      'on':
-      off_html:
+      subject: 您已更改 RubyGems.org 的電子郵件通知設定
+      title: 電子郵件通知
+      subtitle: 嗨 %{handle}
+      'on': "開"
+      off_html: <b>關</b>
     gem_pushed:
-      subject:
-      title:
+      subject: Gem %{gem} 已推送至 RubyGems.org
+      title: Gem 已推送
     gem_yanked:
-      subject:
-      title:
+      subject: Gem %{gem} 已從 RubyGems.org 移除
+      title: Gem 已移除
     reset_api_key:
-      subject:
-      title:
-      subtitle:
+      subject: RubyGems.org 上的 API 金鑰遭重設
+      title: API 金鑰重設
+      subtitle: 嗨 %{handle}
     webauthn_credential_created:
       subject:
-      title:
-      subtitle:
+      title: 已新增安全裝置
+      subtitle: 嗨 %{handle}！
     webauthn_credential_removed:
       subject:
-      title:
-      subtitle:
+      title: 安全裝置已移除
+      subtitle: 嗨 %{handle}！
     totp_enabled:
       subject:
-      title:
-      subtitle:
+      title: 已啟用驗證器應用程式
+      subtitle: 嗨 %{handle}！
     totp_disabled:
       subject:
-      title:
-      subtitle:
+      title: 已停用驗證器應用程式
+      subtitle: 嗨 %{handle}！
     email_reset_update:
       subject:
       title:
     ownership_confirmation:
-      subject:
-      title:
-      subtitle:
-      body_text:
+      subject: 請確認 RubyGems.org 上 %{gem} Gem 的所有權
+      title: 所有權確認
+      subtitle: 嗨 %{handle}！
+      body_text: %{authorizer} 把您加入了 %{gem} Gem 的擁有者名單。請點擊下方連結來確認您的所有權。
```

這行似乎有問題，包個雙引號看看？
```suggestion
      body_text: "%{authorizer} 把您加入了 %{gem} Gem 的擁有者名單。請點擊下方連結來確認您的所有權。"
```

error message:
```
I18n::InvalidLocaleData: can not load translations from /home/runner/work/rubygems.org/rubygems.org/config/locales/zh-TW.yml: #<Psych::SyntaxError: (<unknown>): found character that cannot start any token while scanning for the next token at line 262 column 18>
```